config.lua

```diff
@@ -32,7 +32,7 @@ Config.MajorArmoredBleedChance = 45 -- The chance, in percent, that a player wil
 Config.MaxInjuryChanceMulti = 3 -- How many times the HealthDamage value above can divide into damage taken before damage is forced to be applied
 Config.DamageMinorToMajor = 35 -- How much damage would have to be applied for a minor weapon to be considered a major damage event. Put this at 100 if you want to disable it
 Config.AlertShowInfo = 2 -- How many injuries a player must have before being alerted about them
-
+Config.BleedoutInterval = 50
```

Comment:
A comment next to this config item wouldn't be a bad thing :)

Reply:
fixed this! this was actually vital for this note i myself couldnt remember what variable to change to change this interval haha